src/main/java/com/employeesService/EmployeesService/model/Department.java

```diff
@@ -1,9 +1,27 @@
 package com.employeesService.EmployeesService.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity(name = "departments")
```

Comment:
table name should be `departments`, not the Entity name, you've mistaken it with `@Table(name = "")`